Automattic/wp-calypso
# Guided Onboarding: Link blog_id with survey segmentation data

Related to https://github.com/Automattic/dotcom-forge/issues/8038

## Proposed Changes

* Add `has_segmentation_survey` param on site creation to link the segmentation survey to the newly created blog.

## Why are these changes being made?


* These changes are being made to ensure the `blog_id` is correctly associated with survey segmentation data in the DB.

## Testing Instructions



* Checkout branch or use live link
* Navigate to `/start/guided`
* Go through the flow and verify the `has_segmentation_survey` option is added to site creation

## Pre-merge Checklist
